buildpack.toml

```diff
@@ -13,3 +13,8 @@ id = "*"
 
 [[buildpack.licenses]]
 type = "BSD-3-Clause"
+
+[metadata]
+[metadata.release]
+[metadata.release.docker]
```

Comment:
It would be good to rename this: https://github.com/heroku/languages-github-actions/issues/45